Resources/Computing/ComputingElementFactory.py

```diff
@@ -58,11 +58,13 @@ def getCE(self, ceType='', ceName='', ceParametersDict={}):
       ceDict = {'CEType': ceTypeLocal}
       if ceParametersDict:
         ceDict.update(ceParametersDict)
-      computingElement.setParameters(ceDict)
+      result = computingElement.setParameters(ceDict)
+      if not result['OK']:
+        return result
+
     except BaseException as x:
-      msg = 'ComputingElementFactory could not instantiate %s object: %s' % (subClassName, str(x))
-      self.log.exception()
-      self.log.warn(msg)
+      msg = 'ComputingElementFactory could not instantiate %s object' % subClassName
+      self.log.exception(msg)
```

Comment:
Why? Honestly I prefer how it was written before.

Reply:
I thought the `warn` was unnecessary. Why do you prefer it like this?
I canceled my change.